notebooks/2.mutation-classifier.ipynb

```diff
@@ -4,7 +4,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Create a logistic regression model to predict mutation from gene expression data in TCGA"
+    "# Predicting mutations from gene expression using TCGA PanCanAtlas\n",
+    "\n",
+    "This notebook trains a machine learning model to predict mutations from gene expression data. We use gene expression data as an overall measurement of the \"state\" of the system, and we hypothesize that a machine learning model can learn to distinguish malfunctioning genes by the system's downstream response.\n",
+    "\n",
+    "The model is an elastic net penalized logistic regression classifier and learns a series of weights (or gene importance scores) that together can help predict when genes are altered in a tumor. The notebook loads PanCanAtlas data, trains a machine learning model, and visualizes performance. The notebook is targeted for research purposes and can be used as a jumping-off point to generate new hypotheses. For more details about our approach see [Way _et al._ 2018](https://doi.org/10.1016/j.celrep.2018.03.046).\n",
```

Comment:
Would be helpful to put article title in link (as href tile)

```
[Way et al. 2018](https://doi.org/10.1016/j.celrep.2018.03.046 "tile goes here")
```
I don't know what title is because DOI does not resolve yet.

Also I removed italics, as I think et al shouldn't be italicized as per https://english.stackexchange.com/a/180547